ConorMFinn/CO600-SeriousCyberSec
# Created Case 4

**Description of work.**
Added level four to the game
Organise the file system for scenes to make them easier to navigate
Fix some recurring errors in the scripts

**To test:**
1. Answer all questions
2. Check for typos
3. Make sure interactions are working properly

Fixes #134
